src/app-loader.tsx

```diff
@@ -99,6 +99,62 @@ export const AppLoader = ({ children }: { children: React.ReactElement }) => {
     }
   }, [appDispatch, ethAddress, vegaKeysLoaded, setAssociatedBalances]);
 
+  React.useEffect(() => {
+    let interval: any = null;
+
+    const getNetworkLimits = async () => {
+      const [networkLimits, stats] = await Promise.all([
+        fetch("https://n04.d.vega.xyz/network/limits").then((res) =>
```

Comment:
Hard coded for dev, intentional?

Same on line 110

Reply:
hardcoded just for now. I think Phil is adding it to GQL so waiting on that 😄

Reply:
Ah i didn't realize this was a new API :+1:

Reply:
Updated to use REACT_APP_VEGA_URL with new helper function `getDataNodeUrl`